sidecar-example.yml

```diff
@@ -60,3 +60,18 @@ log_path: "/var/log/graylog-sidecar"
 # TODO: Document or remove depending on the outcome of https://github.com/Graylog2/collector-sidecar/issues/251
 log_rotation_time: 86400
 log_max_age: 604800
+
+# A list of binaries which are allowed to be execute by the Sidecar. An empty list disables the whitelist feature.
+# Example:
+#     collector_binaries_whitelist:
+#       - "/usr/bin/filebeat"
+#       - "/opt/collectors/*"
```

Comment:
Please add one sentence about using wildcards (`*`) so the user cannot miss that feature.